Readme configure elm-analyse chapter (#160)
elm-tooling/elm-language-server: README.md

Settings may need a restart to be applied.

### Elm-Analyse Configuration

> `elm-analyse` is provides for static code analysis. All checks are enabled by default. An (optional) `elm-analyse.json` configuration file will be respected, if added to the project.

Reviewer: ```suggestion
`elm-analyse` is used for static code analysis. All checks are enabled by default. An (optional) `elm-analyse.json` configuration file will be respected, if added to your project.
```
